pkgs/applications/graphics/gimp/plugins/default.nix

```diff
@@ -196,18 +196,18 @@ stdenv.lib.makeScope pkgs.newScope (self: with self; {
   lightning = scriptDerivation {
     name = "Lightning";
     src = fetchurl {
-      url = "http://registry.gimp.org/files/Lightning.scm";
+      url = "https://github.com/pixlsus/registry.gimp.org_static/raw/master/registry.gimp.org/files/files/Lightning.scm";
       sha256 = "c14a8f4f709695ede3f77348728a25b3f3ded420da60f3f8de3944b7eae98a49";
     };
   };
 
   /* space in name trouble ?
 
   rainbowPlasma = scriptDerivation {
```

Comment:
Probably time to drop these here as well.

Reply:
I removed rainbowPlasma and lightningGate but would rather not go too far into cleaning up gimpPlugins. I'm currently only interested in maintaining glimpsePlugins, sorry.